docs/tutorial/part-one/index.md

```diff
@@ -48,10 +48,11 @@ To install a starter, first install Gatsby's command line program by running the
 npm install --global gatsby-cli
 ```
 
-Once that's installed, type the following command in and hit ENTER:
+Once that's installed, open a new terminal window and run the following commands to create a new Gatsby site in a directory called tutorial-part-one and then move to this new directory:
```

Comment:
Is there a need to “open a new terminal window”? Since we just ran `npm install --global gatsby-cli` the window should be open already, correct?

Reply:
Also, @shannonbux, I’d make the same edit here -- replace “move” with “change"